src/Domain/PasswordResetToken.php

```diff
@@ -43,14 +43,24 @@ class PasswordResetToken
      */
     private ?\DateTimeImmutable $used = null;
 
-    public function __construct(Person $person)
+    private function __construct(Person $person, Uuid $uuid)
     {
         $this->person = $person;
-        $this->secret = Uuid::v4();
+        $this->secret = $uuid;
```

Comment:
@bdsl looks like we have some many names for the same thing. E.g. the password reset token is sometimes referred to as `secret`, `uuid`, `base58`, and `token`. Is there any chance we can unify the names?

Reply:
Fair question. Let me take a look. I think Token and Secret are two slightly different things - the token is the object that includes the info about which user its for, whether it's used or not, when it was created. The secret is just the random number.

But secret, uuid, and base58 are all basically the same thing.

Reply:
`⇧F6` 's done in 386b1c6

Reply:
Nice one!